src/ducks/components/intents/InstallKonnectorIntent.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
```

Comment:
Again, has this component to be specific to konnector? Could we use a generic `InstallAppIntent` or this component is doing some things specific to konnectors?